cmd/kubehound/config.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/DataDog/KubeHound/pkg/cmd"
+	"github.com/DataDog/KubeHound/pkg/telemetry/log"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	configPath string
+)
+
+var (
+	configCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Show the current configuration",
+		Long:  `[devOnly] Spawn the kubehound dev stack for the system-tests (build from dockerfile)`,
+		PreRunE: func(cobraCmd *cobra.Command, args []string) error {
+			return cmd.InitializeKubehoundConfig(cobraCmd.Context(), "", true, true)
+		},
+		RunE: func(cobraCmd *cobra.Command, args []string) error {
+			// Adding datadog setup
+			khCfg, err := cmd.GetConfig()
+			if err != nil {
+				return fmt.Errorf("get config: %w", err)
+			}
+
+			yamlData, err := yaml.Marshal(&khCfg)
+
+			if err != nil {
+				return fmt.Errorf("marshaling khCfg: %v", err)
+			}
+
+			if configPath != "" {
+				f, err := os.Create(configPath)
+				if err != nil {
+					return fmt.Errorf("creating file: %v", err)
+				}
+
+				_, err = f.Write(yamlData)
+				if err != nil {
+					return fmt.Errorf("writing to file: %v", err)
+				}
+
+				log.I.Infof("Configuration saved to %s\n", configPath)
+
+				return nil
+			}
+
+			fmt.Println(" --- YAML ---")  //nolint:forbidigo
```

Comment:
Isn't it confusing that you use `---` since it's also the "start of document" in yaml? But at the same time the rest of the line should fail since it's not valid yaml (and it also has a space prefixed, that might be a typo)?